package.json

```diff
@@ -4,14 +4,52 @@
   "version": "0.0.1",
   "description": "The addon where ideas come to idea",
   "main": "index.js",
-  "author": "Les Orchard <lorchard@mozilla.com>",
+  "author": [
+    "Les Orchard <lorchard@mozilla.com>",
+    "Dave Justice <djustice@mozilla.com"
+  ],
   "engines": {
     "firefox": ">=38.0a1",
     "fennec": ">=38.0a1"
   },
-  "license": "MIT",
+  "scripts": {
+    "start": "st -nc .",
+    "package": "jpm xpi && mv @idea-town-addon-0.0.1.xpi dist/idea-town-addon-0.0.1.xpi"
+  },
+  "license": "MPL-2.0",
+  "dependencies": {
+    "exclude": "^1.0.0",
+    "xtend": "^4.0.0"
+  },
   "devDependencies": {
-    "gulp": "^3.9.0",
-    "gulp-connect": "^2.2.0"
+    "st": "^0.5.5"
+  },
+  "CONFIG": {
```

Comment:
I think we probably want QA to be able to change which server an addon is pointed at. The simple-prefs docs mention using `hidden` to indicate that a pref shouldn't be displayed in the addon manager screen for that addon. I assume the prefs would still be accessible via about:config, but I haven't tested it yet.